rentals.json

```diff
@@ -0,0 +1,30 @@
+[
+  {
+    "json_class": "Rental",
+    "data": {
+      "date": "9-4-22",
+      "book": {
+        "title": "Best Actions",
+        "author": "David"
+      },
+      "person": {
+        "name": "Ahmad",
+        "age": 34
+      }
+    }
+  },
```

Comment:
- [ ] Great job so far @Jawad-Nazari and @Mikey1-nuel However, I noticed that as the previous reviewer mentioned in this comment, your person is now able to belong to rental and saved in the JSON file. However, when I try to retrieve that rental information for a particular person, it returns nothing. :worried: 
- [ ] From the video below, you can see that the data is saved into a JSON file, and the person (`Ahmad`) does belong to a rental created however when I try to view the Rental, it returns nothing. So kindly fix this :wink: 
- [ ] To be extra certain, I created a new rental altogether but I still received the same feedback :+1: 

- Video Reference

[Screencast from 07-05-2023 12:01:17 PM.webm](https://github.com/Jawad-Nazari/School_library/assets/102699958/9f94d915-3bc5-4b9e-9049-debb1b1d003f)

Reply:
Hi @NahnahAJ ,
Thanks for your time and and for reviewing our code. We will come up with a solution and fix this issue. 🙏